File: Day-40/BubbleSort.py
def BubbleSort(arr):
    for i in range(0, len(arr)):
        swapped = False
        for j in range(0, len(arr)-i-1):
            if(arr[j]>arr[j+1]):
                temp= arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
                swapped =True
                
        if (swapped == False):
            break
    print(arr)

BubbleSort([10, 1, 7, 6, 14, 9]);

File: Day-40/BubbleSort.js
function BubbleSort(arr) {
  for (let i = 1; i < arr.length; i++) {
    let swap = false;
    for (let j = 0; j < arr.length - i; j++) {
      if (arr[j] > arr[j + 1]) {
        let temp = arr[j];
        arr[j] = arr[j + 1];
        arr[j + 1] = temp;
        swap = true;
      }
    }
    if (swap == false) {
      break;
    }
  }
  console.log(arr);
}

BubbleSort([10, 1, 7, 6, 14, 9]);
